### PR TITLE
Lca adders

### PR DESCRIPTION
**Pull request by m-jamieson**
Updated the generation mix to include MIXED power plants.
Changed the error handling in alt_generation to catch numpy warnings.
Cleaned up sheetname keywords in read_csv calls.

### CODE REVIEW

File: electricitylci/__init__.py [hunk 1]
```diff
@@ -407,7 +407,8 @@ def aggregate_gen(gen_df, subregion="BA"):
         are 'eGRID', 'NERC', 'FERC', 'BA', 'US', by default 'BA', by default "BA"
     """
     import electricitylci.alt_generation as alt_gen
-
+    if subregion is None:
```
Comment (gschivley) [inline electricitylci/__init__.py:428]: Change parameter default to None?

**Comment by gschivley** [comment]
@jump2conclusionsmatt I don't have time to test the code but it generally looks good. Did you confirm that it still works?

File: electricitylci/__init__.py (hunk 1 above)
Comment (m-jamieson) [reply electricitylci/__init__.py:428]: Yeah I'm fine with that.

**Comment by m-jamieson** [comment]
Everything should be fine. I'm actually getting the settings together to run through the code_testing spreadsheet I put up on slack.

**Comment by m-jamieson** [comment]
I should add that the consumption mix calculations will be changed in the next business day or 2. We realized that the FERC consumption mixes should connect to balancing authorities rather than FERC regions to provide a more correct calculation of emissions. Anyhow, it may be worth holding off on the merge until that's done.

**Comment by gschivley** [comment]
I'm going to move my EPA PR from draft to ready for review. Additional changes here will flow through once they're finalized.

**Comment by gschivley** [comment]
I haven't had time to look through but am accepting since it's needed for the large pull to EPA.

**Pull request merged**